### PR TITLE
docs(marble-testing): Edit marble testing

### PR DESCRIPTION
**Pull request by oboukli**
Edit documentation of marble testing and clean up Markdown markup.

### CODE REVIEW

File: docs_app/content/guide/testing/marble-testing.md [hunk 1]
```diff
@@ -2,24 +2,24 @@
 # Testing RxJS Code with Marble Diagrams
 
 <div class="alert is-helpful">
-  <span>This guide refers to usage of marble diagrams when using the new `testScheduler.run(callback)`. Some details here do not apply to using the TestScheduler manually, without using the `run()` helper. </span>
+  <span>This guide refers to usage of marble diagrams when using the new testScheduler.run(callback). Some details here do not apply to using the TestScheduler manually, without using the run() helper.</span>
```
Comment (JWO719) [inline docs_app/content/guide/testing/marble-testing.md]: I personally prefer having the `` consistently used

File: docs_app/content/guide/testing/marble-testing.md [hunk 2]
```diff
@@ -71,7 +71,8 @@ How many virtual milliseconds one frame represents depends on the value of `Test
 - `'|'` complete: The successful completion of an observable. This is the observable producer signaling `complete()`.
 - `'#'` error: An error terminating the observable. This is the observable producer signaling `error()`.
 - `[a-z0-9]` e.g. `'a'` any alphanumeric character: Represents a value being emitted by the producer signaling `next()`. Also consider that you could map this into an object or an array like this:
-```ts
+
+```js
```
Comment (JWO719) [inline docs_app/content/guide/testing/marble-testing.md]: please change back to ts. It should be used all across the docs

File: docs_app/content/guide/testing/marble-testing.md [hunk 3]
```diff
@@ -90,30 +92,28 @@ How many virtual milliseconds one frame represents depends on the value of `Test
...
 - `'^'` subscription point: (hot observables only) shows the point at which the tested observables will be subscribed to the hot observable. This is the "zero frame" for that observable, every frame before the `^` will be negative. Negative time might seem pointless, but there are in fact advanced cases where this is necessary, usually involving ReplaySubjects.
 
 ### Time progression syntax
 
-The new time progression syntax takes inspiration from the CSS duration syntax. It's a number (int or float) immediately followed by a unit; ms (milliseconds), s (seconds), m (minutes). e.g. `100ms`, `1.4s`, `5.25m`.
+The new time progression syntax takes inspiration from the CSS duration syntax. It's a number (integer or floating point) immediately followed by a unit; ms (milliseconds), s (seconds), m (minutes). e.g. `100ms`, `1.4s`, `5.25m`.
```
Comment (JWO719) [inline docs_app/content/guide/testing/marble-testing.md]: int or float is completly correct term here

File: docs_app/content/guide/testing/marble-testing.md [hunk 4]
```diff
@@ -90,30 +92,28 @@ How many virtual milliseconds one frame represents depends on the value of `Test
...
 
 When it's not the first character of the diagram it must be padded a space before/after to disambiguate it from a series of marbles. e.g. `a 1ms b` needs the spaces because `a1msb` will be interpreted as `['a', '1', 'm', 's', 'b']` where each of these characters is a value that will be next()'d as-is.
 
-**NOTE**: You may have to subtract 1 millisecond from the time you want to progress because the alphanumeric marbles (representing an actual emitted value) _advance time 1 virtual frame_ themselves already, after they emit. This can be very unintuitive and frustrating, but for now it is indeed correct.
+**NOTE**: You may have to subtract 1 millisecond from the time you want to progress because the alphanumeric marbles (representing an actual emitted value) _advance time 1 virtual frame_ themselves already, after they emit. This can be counter-intuitive and frustrating, but for now it is indeed correct.
 
-```ts
+```js
```
Comment (JWO719) [inline docs_app/content/guide/testing/marble-testing.md]: also back to ts here

File: docs_app/content/guide/testing/marble-testing.md [hunk 5]
```diff
@@ -204,17 +205,17 @@ it('should repeat forever', () => {
...
+### RxJS code that consumes Promises or uses any of the other schedulers (e.g. AsapScheduler) cannot be directly tested
 
-If you have RxJS code that uses any other form of async scheduling other than AsyncScheduler, e.g. Promises, AsapScheduler, etc. you can't reliably use marble diagrams _for that particular code_. This is because those other scheduling methods won't be virtualized or known to TestScheduler.
+If you have RxJS code that uses any other form of asynchronous scheduling other than `AsyncScheduler`, e.g. Promises, `AsapScheduler`, etc. you can't reliably use marble diagrams _for that particular code_. This is because those other scheduling methods won't be virtualized or known to TestScheduler.
 
-The solution is to test that code in isolation, with the traditional async testing methods of your testing framework. The specifics depend on your testing framework of choice, but here's a pseudo-code example:
+The solution is to test that code in isolation, with the traditional asynchronous testing methods of your testing framework. The specifics depend on your testing framework of choice, but here's a pseudo-code example:
 
-```ts
+```js
```
Comment (JWO719) [inline docs_app/content/guide/testing/marble-testing.md]: ts

**Comment by JWO719** [comment]
Hi @oboukli ,

in the meantime it seems like there are some merge conflicts. I would really like to merge this one. Could you maybe resolve the conflicts and solve the comments above. If you are facing any issues with that, don't hesitate to ask me for help! Thanks again for this contribution, I really appreciate that

File: docs_app/content/guide/testing/marble-testing.md (hunk 4 above)
Comment (oboukli) [reply docs_app/content/guide/testing/marble-testing.md]: Fixed

**Comment by oboukli** [comment]
@JWO719 Please feel free to include me in future doc PR reviews. I'm more happy to assist and contribute.

File: docs_app/content/guide/testing/marble-testing.md (hunk 5 above)
Comment (oboukli) [reply docs_app/content/guide/testing/marble-testing.md]: Fixed

**Pull request merged**